From: Jakub Jirutka <jakub@jirutka.cz>
Date: Thu, 21 Jul 2022 18:28:00 +0200
Subject: [PATCH] Fix build with mupdf on Alpine

Alpine's mupdf-dev provides mupdf.pc.
mupdf >=1.19.0 doesn't build libmupdf-third.

--- a/meson.build
+++ b/meson.build
@@ -19,42 +19,15 @@
 girara = dependency('girara-gtk3')
 glib = dependency('glib-2.0')
 cairo = dependency('cairo')
-mupdf = dependency('mupdf', required: false)
-mupdfthird = cc.find_library('mupdf-third')
+mupdf = dependency('mupdf')
 
 build_dependencies = [
   zathura,
   girara,
   glib,
   cairo,
+  mupdf,
 ]
-
-if not mupdf.found()
-  # normal build of mupdf
-  mupdf = cc.find_library('mupdf')
-  build_dependencies += [mupdf, mupdfthird]
-else
-  # build from Debian's libmupdf-dev
-  build_dependencies += [mupdf, mupdfthird]
-
-  libjpeg = dependency('libjpeg')
-  libjbig2dec = cc.find_library('jbig2dec')
-  libopenjp2 = dependency('libopenjp2')
-  gumbo = dependency('gumbo')
-  tesseract = dependency('tesseract')
-  leptonica = dependency('lept')
-  mujs = dependency('mujs')
-
-  build_dependencies += [
-    libjpeg,
-    libjbig2dec,
-    libopenjp2,
-    gumbo,
-    tesseract,
-    leptonica,
-    mujs
-  ]
-endif
 
 if get_option('plugindir') == ''
   plugindir = zathura.get_pkgconfig_variable('plugindir')
